app/controllers/api/internal/challenges_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Api::Internal::ChallengesController < Api::Internal::BaseController
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Not covered lines: 1 to 39